typed-racket-lib/typed-racket/base-env/require-typed.rkt

```diff
@@ -0,0 +1,432 @@
+#lang racket/base
+
+;; This file is for all prims that use contracts, so that we can lazily load them.
```

Comment:
Can we name this module something different since it seems odd to put forms like `cast` and `make-predicate` in the `require-typed.rkt` file?